Definition for nerd (#267)
tatianamac/selfdefined: 11ty/definitions/nerd.md

---
title: nerd
slug: nerd
flag:
  level: avoid
defined: true
speech: noun
alt_words:
  relating to passion:
    - fan
    - aficionado
    - enthusiast
  relating to skill or intelligence:
    - expert
    - adept
    - specialist
---

> From the [DHH](https://medium.com/dictionary-of-digital-humanities/nerd-b7d61b6f536e): "An overly intellectual person, usually obsessive or lacking social skills, focusing their interests on non-mainstream activities, generally highly technical or related to fiction or fantasy narratives. It is used as a stereotype for either derogatory or praising purposes, depending on who's wielding it."

Reviewer: ```suggestion
"an overly intellectual person, usually obsessive or lacking social skills, focusing their interests on non-mainstream activities, generally highly technical or related to fiction or fantasy narratives. It is used as a stereotype for either derogatory or praising purposes, depending on who's wielding it."
–[Dictionary of Digital Humanities](https://medium.com/dictionary-of-digital-humanities/nerd-b7d61b6f536e)
```

---
title: nerd
slug: nerd
flag:
  level: avoid
defined: true
speech: noun
alt_words:
  relating to passion:

>     - fan
>     - aficionado
>     - enthusiast

Reviewer: ```suggestion
    - aficionado
    - enthusiast
    - fan
```

>   relating to skill or intelligence:
>     - expert
>     - adept
>     - specialist
> ---

Reviewer: ```suggestion
  relating to skill or intelligence:
    - adept
    - expert
    - specialist
---
```

"an overly intellectual person, usually obsessive or lacking social skills, focusing their interests on non-mainstream activities, generally highly technical or related to fiction or fantasy narratives. It is used as a stereotype for either derogatory or praising purposes, depending on who's wielding it."
–[Dictionary of Digital Humanities](https://medium.com/dictionary-of-digital-humanities/nerd-b7d61b6f536e)

## Issues

> The term can be used in a way that leads to weaponizing a person's passion against them, which may manifest as bullying of that person's appearance or background.

Reviewer: ```suggestion
When used in reference to someone else without their consent, the term can weaponise a person's passion against them, or bully someone based on their appearance or background.
```

## Impact

By assigning negativity to the passion that someone exhibits for the things they enjoy, we invalidate their individuality and reduce the enjoyment they derive from their passions. As a result, we may cause these people to abandon hobbies or interests, instilling a feeling of wrongness in finding joy how they see fit (provided their methods of finding joy are not harmful).

> ## Taking It Back
> 
> Individuals can utilize this term in a self-descriptive way to identify that they are extremely passionate about their hobbies and interests. Acknowledging this usage and supporting people who see themselves this way is an effective method of reclaiming the positive aspects of this term and validating the way people see and think of their passions as it relates to their identity.

Reviewer: ```suggestion
## Self-Identification

Individuals may use this term to self-identify and to indicate their extreme passionate for hobbies and interests. Acknowledging this usage and supporting people who self-identify and reclaim this term can validate the way people perceive their passions as it relates to their identity.
```

Reviewer: I'd additionally recommend moving this up immediately before Issues

Author: Can do!
